core/src/test/java/io/questdb/griffin/SqlParserTest.java

```diff
@@ -4766,7 +4766,7 @@ public void testNullChecks() throws SqlException {
                 "SELECT \n" +
                         "a\n" +
                         "FROM x WHERE b = 'H' AND time in('2020-08-01T17:00:00.305314Z' , '2020-09-20T17:00:00.312334Z')\n" +
-                        "select *", // <-- dangling 'select *'
+                        "select * from long_sequence(1)", // <-- dangling 'select * from long_sequence(1)'
```

Comment:
nit: comment does not make sense anymore, nothing dangling anymore. Perhaps remove comment to avoid future confusion.